src/outputs/outputs.cpp

```diff
@@ -263,6 +263,7 @@ Outputs::Outputs(Mesh *pm, ParameterInput *pin, SimTime *tm) {
           num_rst_outputs++;
         }
 #ifdef ENABLE_HDF5
+        op.write_xdmf = pin->GetOrAddBoolean(op.block_name, "write_xdmf", true);
```

Comment:
Add to the doc that his param now exists?

Reply:
:+1: done